README.md

```diff
@@ -37,7 +37,6 @@ Our [Get Started guide](https://docs.kedro.org/en/stable/get_started/install.htm
 
 ## What are the main features of Kedro?
 
-![Kedro-Viz Pipeline Visualisation](https://github.com/kedro-org/kedro-viz/blob/main/.github/img/banner.png)
```

Comment:
I'm sad about seeing this image go though, it's arguably one of the things that hook people to Kedro 😇 I know it's technically part of Kedro Viz but if we consider Framework as the center of the ecosystem, maybe it makes sense to keep it?

Reply:
OK, I can reinstate but I will reorganise the content slightly as it wasn't in the right place in the narrative.

I think I have a problem with these graphics as they're very dark and not the easiest to read/interpret but if there's evidence they encourage uptake, it's probably just me, and let's keep it.